src/main/cpp/SDLPlayData.cpp

```diff
@@ -881,30 +886,31 @@ void SDLPlayData::event_loop(VideoState *is) {
 				incr = -60.0;
 			do_seek:
 				//TODO FIX SEEK BY BYTES BUG
-				//if (seek_by_bytes) {
-				//	pos = -1;
-				//	if (pos < 0 && is->get_video_stream() >= 0)
-				//		pos = is->get_pPictq()->last_pos();
-				//	if (pos < 0 && is->get_audio_stream() >= 0)
-				//		pos = is->get_pSampq()->last_pos();
-				//	if (pos < 0)
-				//		pos = avio_tell(is->get_ic()->pb);
-				//	if (is->get_ic()->bit_rate)
-				//		incr *= is->get_ic()->bit_rate / 8.0;
-				//	else
-				//		incr *= 180000.0;
-				//	pos += incr;
-				//	is->stream_seek(pos, incr, 1);
-				//}
-				//else {
+				seek_by_bytes = 0;
```

Comment:
Please use the original code from above commented out and add a 
TODO: Fix seek by byte; check if this worked in the original ffplay.c first

Reply:
Can't make the seek by byte work with ffplay, I have changed the initial value of seek_by_byte to false to disable it for now